src/group_theory/nilpotent.lean

```diff
@@ -55,6 +55,7 @@ subgroup `G` of `G`, and `⊥` denotes the trivial subgroup `{1}`.
   `least_descending_central_series_length_eq_nilpotency_class` and
   `lower_central_series_length_eq_nilpotency_class`.
 * If `G` is nilpotent, then so are its subgroups, images, quotients and preimages.
+  Finite products of nilpotent groups are nilpotent.
```

Comment:
```suggestion
  Binary products of nilpotent groups are nilpotent.
```
I'd save your phrasing for the statement:
```lean
lemma is_nilpotent_pi {ι : Type*} (G : ι → Type*) [fintype ι]
  [∀ i, group (G i)] [∀ i, is_nilpotent (G i)] : is_nilpotent (Π i, G i) :=
sorry
```
(which you can always add in a follow-up PR if you like 😉)

Reply:
You mean this PR? :-)
https://github.com/leanprover-community/mathlib/pull/11829